docs/machineset-gcp.md

```diff
@@ -0,0 +1,74 @@
+# Creating a GCP Windows MachineSet
+
+_\<windows_server_image \>_ should be replaced with the full path to an image of a [supported version](wmco-prerequisites.md#supported-windows-server-versions) of Windows.
+For example, to use the latest image from the `windows-2022-core` family in the `windows-cloud` project, you would use:
+`projects/windows-cloud/global/images/family/windows-2022-core`
+
+
+_\<infrastructure_id\>_ should be replaced with the output of:
+```shell script
+ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
+```
+_\<region\>_ should be replaced with a region, such as `us-central1`.
+
+_\<zone\>_ should be replaced with a zone within the chosen region, such as `us-central1-a`.
+
+_\<zone_suffix\>_ should be replaced with the suffix of the chosen zone, such as `a`.
+
+_\<project_id\>_ should be replaced with the GCP project this cluster was created under.
```

Comment:
Is there a `gcloud`  command to fetch this info? `gcloud config get-value project` maybe?

Reply:
This could give the wrong value if the default project they have set up on their shell isn't the one that the cluster exists on

Reply:
I follow, is it safe to use it in the [machineset.sh](https://github.com/openshift/windows-machine-config-operator/blob/258b568146e2e7e2637aa0c28a0062bf34f6d20b/hack/machineset.sh#L189) script?

Reply:
i think its fine to use there as that script is meant for our use, not for customers